axolotl-web/src/App.vue

```diff
@@ -1,56 +1,56 @@
 <template>
   <div id="app" @click="checkDebug">
-    <header-comp></header-comp>
+    <header-comp />
     <main class="container">
-      <error-modal  v-if="error"/>
+      <error-modal v-if="error" />
       <router-view />
     </main>
   </div>
 </template>
 
 <script>
-window.getCookie = function(cname) {
-      var name = cname + "=";
-      var ca = document.cookie.split(';');
-      for(var i = 0; i < ca.length; i++) {
-        var c = ca[i];
-        while (c.charAt(0) == ' ') {
-          c = c.substring(1);
-        }
-        if (c.indexOf(name) == 0) {
-          return c.substring(name.length, c.length);
-        }
-      }
-      return false;
-}
-if (window.getCookie("darkMode") === 'true') {
-  import ('./assets/dark.scss');
+window.getCookie = function (cname) {
+  var name = cname + "=";
```

Comment:
Call me modern, but as I understand it, we do not need to limit ourselves when it comes to more modern JavaScript features, right? I would argue that `var` nowadays is basically replaced by `let` and `const`. The go `const` and the JavaScript `const` means the same. `let` is basically the modern `var`.

In this case, we could replace `name` and `ca` with const, and the variables inside the loop with `let`.

Here an article about let and const:
https://appdividend.com/2020/08/28/javascript-const-vs-let-the-complete-guide/

Reply:
i will respect the `const` and `let` in the future, thanks for the hint.